src/modules/aws_acm/rpcs/import.ts

```diff
@@ -40,6 +40,13 @@ export class CertificateImportRpc extends RpcBase {
     message: 'varchar',
   } as const;
 
+  inputTable: RpcInput = [
```

Comment:
currently the documentation for rpcs was quite manual, explaining input/outputs just in manual comments. Have you tried adding comments at this level to see how they display, and if they may be more informative than what we have now?

Reply:
I'll investigate that as a follow-up. Is that ok?

Reply:
sure